Unlicense

```diff
@@ -0,0 +1,24 @@
+This is free and unencumbered software released into the public domain.
```

Comment:
Maybe Grakkit should use a MIT license instead?

Reply:
Why's this? Spacey said that he thinks the Unlicense is the best fit, but if you can give other reasons why not to use it, I'm all for it

Reply:
Ah fair enough, that's a reasonable concern